packages/core/src/interfaces/charts.ts

```diff
@@ -137,7 +138,14 @@ export interface BaseChartOptions {
 		 * options related to gradient
 		 * e.g. { enabled: true }
 		 */
-		gradient?: object;
+		gradient?: {
+			enabled?: boolean;
+			/**
+			 * hex color array
+			 * e.g. ['#fff', '#000', ...]
+			 */
+			colors?: Array<String>;
```

Comment:
do we want this applied to other chart types?

Reply:
Yes, I think it makes sense to define what object attribute it accepts, setting it to `object` is not documentation friendly.

[https://charts.carbondesignsystem.com/documentation/interfaces/_interfaces_charts_.basechartoptions.html#color](https://charts.carbondesignsystem.com/documentation/interfaces/_interfaces_charts_.basechartoptions.html#color)

That being said, not all chart types support gradient.